account_easy_reconcile/easy_reconcile.py

```diff
@@ -0,0 +1,424 @@
+                            'easy_reconcile_id': rec.id,
+                            'date': fields.datetime.now(),
+                            'reconcile_ids': [
+                                (4, rid) for rid in reconcile_ids
+                                ],
+                            'reconcile_partial_ids': [
+                                (4, rid) for rid in partial_ids
+                                ],
+                        })
+                except Exception as e:
+                    # In case of error, we log it in the mail thread, log the
+                    # stack trace and create an empty history line; otherwise,
+                    # the cron will just loop on this reconcile task.
+                    _logger.exception(
+                        "The reconcile task %s had an exception: %s",
+                        rec.name, e.message
+                    )
+                    message = "There was an error during reconciliation : %s" \
+                        % e.message
+                    rec.message_post(body=message)
+                    self.env['easy.reconcile.history'].create(
+                        {
+                            'easy_reconcile_id': rec.id,
+                            'date': fields.datetime.now(),
+                            'reconcile_ids': [],
+                            'reconcile_partial_ids': [],
+                        }
+                    )
+                finally:
+                    if ctx['commit_every']:
+                        new_cr.commit()
+                        new_cr.close()
+
+#                     self.env.cr.close()
+
+        return True
+
+    @api.model
+    def _no_history(self, rec):
+        """ Raise an `orm.except_orm` error, supposed to
+        be called when there is no history on the reconciliation
+        task.
+        """
+        raise except_orm(
+            _('Error'),
+            _('There is no history of reconciled '
+              'items on the task: %s.') % rec.name)
+
+    @api.model
+    def _open_move_line_list(self, move_line_ids, name):
+        return {
+            'name': name,
+            'view_mode': 'tree,form',
+            'view_id': False,
+            'view_type': 'form',
+            'res_model': 'account.move.line',
+            'type': 'ir.actions.act_window',
+            'nodestroy': True,
+            'target': 'current',
+            'domain': unicode([('id', 'in', move_line_ids)]),
+        }
+
+    @api.multi
+    def open_unreconcile(self):
+        """ Open the view of move line with the unreconciled move lines"""
+        self.ensure_one()
+        obj_move_line = self.env['account.move.line']
+        line_ids = obj_move_line.search(
+            [('account_id', '=', self.account.id),
+             ('reconcile_id', '=', False),
+             ('reconcile_partial_id', '=', False)])
+        name = _('Unreconciled items')
+        return self._open_move_line_list(line_ids and line_ids.ids or [], name)
```

Comment:
getting `ids` from empty record set won't raise an error. You can write `recs.ids or []`